package.json

```diff
@@ -28,38 +28,38 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.1.2"
+    "ember-cli-babel": "^7.11.1",
+    "ember-cli-htmlbars": "^4.0.0"
```

Comment:
This is not correct I think...

Reply:
Apparently, this is new intended behaviour => https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/index.js#L54

Reply:
I don't think we need it in there though? This addon doesn't shit any templates, right? Isn't this just a "sane default" since most addons will ship components?

Reply:
Removed it.